crates/benches/browser/wasm/Cargo.toml

```
@@ -2,7 +2,7 @@
 edition = "2021"
 name = "tlsn-benches-browser-wasm"
 publish = false
-version = "0.0.0"
+version = "0.0.1"
```

```suggestion
version = "0.0.0"
```

We do use version "0.0.0" for packages with "publish = false"